core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -144,6 +144,13 @@ default void putGcCandidates(TableId tableId, Collection<StoredTabletFile> candi
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Unlike {@link #putGcCandidates(TableId, Collection)} this takes file and dir GC candidates.
+   */
+  default void putGcFdCandidates(TableId tableId, Collection<String> candidates) {
+    throw new UnsupportedOperationException();
+  }
+
```

Comment:
What is "Fd" short for? Should comment what the "Fd" in the method name means.

Reply:
If "Fd" means "file and Directories"  you could just put that explicitly in the name like `putGcFileCandidates` or `putGcFileDirectories`

Reply:
Yeah Fd meant file and dirs.  I can change it to FileOrDirs.  The existing method is more strongly typed, so it probably does not need File in the name.

Reply:
I changed the name in d259836